plutus-core/cost-model/data/models.R

```diff
@@ -531,8 +514,13 @@ modelFun <- function(path) {
         m <- lm(t ~ y_mem, filtered)
         adjustModel(m,fname)
     }
-    
 
+    ## All of the arguments of VerifyEcdsaSecp256k1Signature are of fixed size.
```

Comment:
wait, I thought you were implying that the model should be linear now?

Reply:
Or is that what's going to happen in the other PR?

Reply:
That's talking about `EcdsaSecp256k1`. It's `Ed25519` that's problematic.